README.md

```diff
@@ -15,13 +15,13 @@ Supported on Windows, OSX, Ubuntu and Red Hat.  Written for the .NET Core CLR as
 |![macOS](docs/res/apple_med.png) **macOS**|[![Build & Test][macOS-build-badge]][macOS-build]| 
 |![Linux](docs/res/linux_med.png) **Linux**|[![Build & Test][linux-build-badge]][linux-build]|
 
-[win-build-badge]: https://mseng.visualstudio.com/pipelinetools/_apis/build/status/VSTS.Agent.CI-Windows?branchName=master
+[win-build-badge]: https://mseng.visualstudio.com/pipelinetools/_apis/build/status/azure-pipelines-agent.ci-?branchName=master
```

Comment:
I don't see that definition, should it be azure-pipleines-agent.ci-windows ?

Reply:
https://dev.azure.com/mseng/PipelineTools/_apis/build/status/VSTS.Agent/azure-pipelines-agent.ci-windows?branchName=master
Now that branding is updated, we can also flip over to dev.azure.com/mseng

Reply:
It's there:  https://dev.azure.com/mseng/PipelineTools/_build?definitionId=6916

Reply:
I'll update the urls

Reply:
fixed the missing -windows and tested all the urls